package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neolution-ch/react-pattern-ui",
-  "version": "2.10.0",
+  "version": "2.10.1-alpha.0",
```

Comment:
I guess this version change should be reverted before merging right?

Reply:
it shouldn't matter, the version is completely handled by the action, so as soon as we merge and deploy a major version, it will be automatically changed to "3.0.0"